src/objective-c/tests/UnitTests/GRPCCallOptionsTests.m

```diff
@@ -0,0 +1,161 @@
+/*
+ *
+ * Copyright 2022 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#import <XCTest/XCTest.h>
+
+#import <GRPCClient/GRPCCallOptions.h>
+
+static NSString *const kGRPCCallOptionsTestServerAuthority = @"dummy";
+static NSTimeInterval kGRPCCallOptionsTestTimeout = 11;
```

Comment:
It looks like all the variables here of type `NSString *` are declared `const` but all the non-pointer types are not declared `const`. I strongly recommend also declaring the non-pointer types as `const` so that all these values are constants.

Reply:
will follow up in a cleanup PR